packages/lesswrong/components/ea-forum/wrapped/EAForumWrapped2023Page.tsx

```diff
@@ -0,0 +1,1501 @@
+      const Component = eaEmojiPalette.find(emoji => emoji.label === next.name)?.Component
+      if (Component) {
+        // only go to 96% to prevent causing a horizontal scroll
+        range(0, next.count).forEach(_ => prev.push({
+          top: `${Math.random() * 96}%`,
+          // top: `${(Math.random() + Math.random()) * 96 / 2}%`,
+          left: `${Math.random() * 96}%`,
+          // left: `${(Math.random() + Math.random()) * 96 / 2}%`,
+          Component
+        }))
+      }
+      return prev
+    }, [])
+  }
+
+  const [allReactsReceived, setAllReactsReceived] = useState<ReceivedReact[]>([])
+  useEffect(() => {
+    setAllReactsReceived(placeBackgroundReacts(receivedReacts))
+  }, [receivedReacts])
+  
+  const totalReactsReceived = receivedReacts.reduce((prev, next) => prev + next.count, 0)
+  if (totalReactsReceived <= 5) return null
+  
+  return <section className={classNames(classes.section, classes.sectionTall)}>
+    {allReactsReceived?.map((react, i) => {
+      return <div
+        key={i}
+        className={classes.backgroundReact}
+        style={{top: react.top, left: react.left}}
+      >
+        <react.Component />
+      </div>
+    })}
+    <div className={classes.reactsReceivedContents}>
+      <h1 className={classes.heading3}>
+        Others gave you{" "}
+        <span className={classes.highlight}>
+          {receivedReacts[0].count} {receivedReacts[0].name}
+        </span>{" "}
+        reacts{receivedReacts.length > 1 ? '...' : ''}
+      </h1>
+      {receivedReacts.length > 1 && <div className={classes.otherReacts}>
+        <p className={classes.heading5}>... and {totalReactsReceived} reacts in total:</p>
+        <div className={classNames(classes.stats, classes.mt26)}>
+          {receivedReacts.slice(1).map(react => {
+            return <article key={react.name} className={classes.stat}>
+              <div className={classes.heading3}>{react.count}</div>
+              <div className={classes.statLabel}>{react.name}</div>
+            </article>
+          })}
+        </div>
+      </div>}
+    </div>
+  </section>
+}
+
+/**
+ * Section that displays some recommended posts to the user
+ */
+const RecommendationsSection = ({classes}: {
+  classes: ClassesType
+}) => {
+  return <section className={classes.section}>
+    <h1 className={classes.heading3}>
+      Posts you missed that we think you’ll enjoy
+    </h1>
+    {/* This postId is my April Fool's post. I think it shouldn't affect the results, but it's still required for some reason. */}
+    <Components.RecommendationsList
+      algorithm={{strategy: {name: 'bestOf', postId: 'KPijJRnQCQvSu5AA2'}, count: 5, disableFallbacks: true}}
+      ListItem={
+        (props: {
+          post: PostsListWithVotesAndSequence,
+          translucentBackground?: boolean,
+        }) => (
+          <Post post={props.post} classes={classes} />
+        )
+      }
+      className={classes.recommendedPosts}
+    />
```

Comment:
Let me know if there's something else I should do here, I added this pretty quickly and it seems to work

Reply:
The post id is used in the analytics - you could just set it to "wrapped2023" or something. If you don't want to do that then we should pass it in as the `context` property of the strategy.